src/status_im/common/scroll_page/style.cljs

```diff
@@ -58,15 +58,16 @@
 (def picture-border-width 4)
 
 (defn display-picture-container
-  [animation]
+  [animation theme]
   (reanimated/apply-animations-to-style
    {:transform [{:scale animation}]}
-   {:border-radius picture-diameter
-    :border-width  picture-border-width
-    :border-color  (colors/theme-colors colors/white colors/neutral-95)
-    :position      :absolute
-    :top           (- (+ picture-radius picture-border-width))
-    :left          (+ (/ picture-radius 2) picture-border-width)}))
+   {:border-radius    picture-diameter
+    :border-width     picture-border-width
+    :border-color     (colors/theme-colors colors/white colors/neutral-95)
+    :background-color (colors/theme-colors colors/white colors/neutral-95 theme)
```

Comment:
hi @J-Son89, this use case looks a little inconsistent to me. The `theme-colors` function works even if we don't pass the theme param, as the global theme will be used.
So should we only pass the theme when we are overwriting the global theme, like in a shell, or always pass (though we might not have access to the theme param)? wdyt?

Reply:
(unrelated to pr)

Reply:
I believe we should always pass the 3rd param. Unfortunately that function theme-colors was created with the ratom referenced inside it and it was harder to refactor all older uses/ prevent people from not passing it. 
Imo in the ideal case the page theme (from context) or global theme (from ratom) would be passed as a param and we keep theme-color function pure.
Theme should be accessible all places within the app so there's no issue with that, one exception is the status bar for navigation but we will have to look for a solution to that.
Cc @ilmotta 

@jo-mut can you update the uses here to pass theme along?